js/models/Settings.js

```diff
@@ -26,10 +30,40 @@ export default class extends BaseModel {
   nested() {
     return {
       shippingAddresses: ShippingAddresses,
-      smtpSettings: SMTPSettings,
+      appearanceSettings: AppearanceSettings,
+      transactionSettings: TransactionSettings,
```

Comment:
I'm not seeing transactionSettings in the Settings API:
https://github.com/OpenBazaar/openbazaar-go/blob/master/repo/settings.go

Is that something that still has to be done on the back-end? If so, please keep in mind that it only needs to be a nested model is the server has it as nested data. If the server has the properties with the TransactionSettings model as non-nested attributes, then we should follow suit and integrate them directly into the Settings model (no nesting).